src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,64 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tabs';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  const { content } = selectedTab;
+  const handleClick = (tab: Tab) => {
+    if (selectedTabId !== tab.id) {
+      return onTabSelected(tab);
+    }
+
+    return null;
+  };
```

Comment:
you can write these conditions inside a function in App Component to avoid copying the similar func.